website/src/theme/DocPage/Layout/Main/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Main from '@theme-original/DocPage/Layout/Main'
+import Giscus from '@giscus/react'
+import { useColorMode } from '@docusaurus/theme-common'
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext'
+
+export default function MainWrapper({ children, ...props }) {
+  const { isDarkTheme } = useColorMode()
+  const { siteConfig } = useDocusaurusContext()
+
+  return (
+    <Main {...props}>
+      {children}
+      <div style={{ height: '32px' }} />
+      <div className='row'>
```

Comment:
nitpick: why not rely on Docusaurus' design language?

```suggestion
      <div className='row margin-top--md'>
```

https://infima.dev/docs/layout/spacing/

Reply:
Changed